sci-mathematics/fann/fann-9999.ebuild

```diff
@@ -0,0 +1,32 @@
+EAPI=5
```

Comment:
I'm confused. repoman only complains, it don't fix anything. So I simply copy a header from fann-2.2.0.ebuild...